merge_operator_test.go

```diff
@@ -6,6 +6,16 @@ import (
 	"github.com/facebookgo/ensure"
 )
 
+// fatalAsError is used as a wrapper to make it possible to use ensure
+// also if C calls Go otherwise it will throw a internal lockOSThread error.
+type fatalAsError struct {
```

Comment:
Is it used anywhere?

Reply:
yes just below in this file (https://github.com/DataDog/gorocksdb/pull/28/files#diff-f22ce3b505e233e60c77c10b7ff5c1ba252b2051fa211860ace6c0ac174c05d0R28) 

It was previously in filter_policy_test.go which was deleted (that functionality was deprecated/removed) so I just moved this function close to the other place it was used.

Reply:
Deleted from here: https://github.com/DataDog/gorocksdb/pull/28/files#diff-e615445bfd088ec2c79e68020be56f9c138f2fa2b3aa9a4305b72dd3fe44cd26L11-L17